tests/docker/tox.ini

```diff
@@ -32,8 +32,9 @@ commands =
     docker-compose rm -v -f
     docker-compose up -d flask_app
     docker-compose stop flask_app
```

Comment:
I don't seem to get this code. We up a container and then immediately stop it, what is the point? Is that we expect the test to run in between? That could lead to race conditions.

Reply:
agreed! will  test and validate

Reply:
removed this test case